src/main/java/com/example/bookstore/model/Role.java

```diff
@@ -0,0 +1,35 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+@Data
+@Entity
```

Comment:
it is not recommended to use `@Data` and `@Entity` but we can leave this case as it is since there are no Lazy fields
[more about why](https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/)